src/assemble_workflow/dists.py

```diff
@@ -0,0 +1,24 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+
+from assemble_workflow.dist import Dist, DistTar, DistZip
+
+
+class Dists:
+    DISTRIBUTIONS_MAP = {
```

Comment:
We now have two maps to maintain, here and in BundleRecorder - perhaps add a class method on DistTar/Zip for fetching the extension and import this map into BundleRecorder?

Reply:
Will try that method thanks for advice.

Reply:
I will create a secondary PR after this one to implement this, already have clues.

Thanks.

Reply:
https://github.com/opensearch-project/opensearch-build/pull/1676